templates/textinput.jsx

```diff
@@ -1,7 +1,10 @@
+import Adapt from 'core/js/adapt';
 import React from 'react';
 import { compile, classes, templates, html } from 'core/js/reactHelpers';
 
 export default function TextInput (props) {
+  const ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
```

Comment:
`_globals` should be in the props already?

Reply:
Ah, yep.  Good call.